src/main/java/uk/gov/companieshouse/acsp/service/PaymentsService.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.companieshouse.acsp.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.acsp.Exception.ServiceException;
+import uk.gov.companieshouse.acsp.sdk.ApiClientService;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.handler.payment.request.PaymentCreate;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.model.payment.PaymentApi;
+import uk.gov.companieshouse.api.model.payment.PaymentSessionApi;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.environment.EnvironmentReader;
+import uk.gov.companieshouse.environment.impl.EnvironmentReaderImpl;
+
+
+import java.io.IOException;
+
+@Service
+public class PaymentsService {
+    private final ApiClientService apiClientService;
+
+    @Autowired
+    public PaymentsService(ApiClientService apiClientService) {
+        this.apiClientService = apiClientService;
+    }
+
+    private static EnvironmentReader environmentReader = new EnvironmentReaderImpl();
+
+    public PaymentApi createPaymentSession(String passThroughHeader, Transaction transaction) throws ServiceException {
+//      below is the dummy body, resource should be the payment link generated by transactcion api
+        PaymentSessionApi paymentSessionApi = new PaymentSessionApi();
+        paymentSessionApi.setRedirectUri("https://cidev.aws.chdev.org/transactions/117524-754816-491724/company/10000025/confirmation");
```

Comment:
I am guessing these value will depend on the environment so we need to use variables.

Reply:
addressed the comments